src/Commands/Upstream/Updates/ListCommand.php

```diff
@@ -12,7 +12,7 @@
 class ListCommand extends UpdatesCommand
 {
     /**
-     * Displays a list of new code commits available from the upstream for a site's Dev environment.
+     * Displays a list of new code commits available from the upstream for a site's development environment.
```

Comment:
I like that this is grammatically correct, but the dashboard calls it 'Dev' and the id/name is 'dev', so I think we should keep the nomenclature consistent.